assets/js/modules/adsense/components/module/ModuleTopEarningPagesWidget/Table.js

```diff
@@ -33,12 +33,30 @@ import ReportTable from '../../../../../components/ReportTable';
 import TableOverflowContainer from '../../../../../components/TableOverflowContainer';
 import Link from '../../../../../components/Link';
 import { MODULES_ANALYTICS, DATE_RANGE_OFFSET } from '../../../../analytics/datastore/constants';
+import { MODULES_ADSENSE } from '../../../datastore/constants';
```

Comment:
Per our "best practices" we should use `STORE_NAME` for code (e.g. components) that is part of the same module as the relevant store. In other words, you're editing an AdSense component here, so when referring to the AdSense store, it should use `STORE_NAME`. Only when referring to other stores, the more expressive constant should be used (as you correctly do here for `MODULES_ANALYTICS`.